packages/kotti-ui/source/kotti-navbar/components/NavbarQuickLink.vue

```diff
@@ -1,53 +1,74 @@
 <template>
-	<div class="kt-navbar-quick-links">
-		<div v-if="!isNarrow" class="kt-navbar-quick-links__title" v-text="title" />
+	<div class="kt-navbar__quick-links">
+		<div
+			v-if="!isNarrow"
+			class="kt-navbar__quick-links__title"
+			v-text="title"
+		/>
 		<a
 			v-for="(item, index) in links"
 			:key="index"
-			class="kt-navbar-quick-links__link"
+			class="kt-navbar__quick-links__link"
 			:href="item.link"
 			rel="noopener noreferrer"
 			target="_blank"
 		>
-			<span v-if="!isNarrow" v-text="item.title" /> <i class="yoco">link</i>
+			<span v-if="!isNarrow" v-text="item.title" />
+			<i class="yoco" v-text="Yoco.Icon.LINK" />
 		</a>
 	</div>
 </template>
 
-<script>
-export default {
+<script lang="ts">
+import { Yoco } from '@3yourmind/yoco'
+import { computed, defineComponent } from '@vue/composition-api'
+
+import { useTranslationNamespace } from '../../kotti-translation/hooks'
+import { Kotti } from '../../types'
+
+export default defineComponent<{
+	links: Kotti.Navbar.QuickLink[]
+}>({
 	name: 'KtNavbarQuickLink',
 	props: {
-		title: { type: String, default: 'Quick Links' },
-		links: { type: Array, default: () => [] },
+		links: { required: true, type: Array },
 	},
-	computed: {
-		isNarrow() {
-			return this.$KtNavbar.isNarrow
-		},
+	setup(props, { root }) {
+		const translations = useTranslationNamespace('KtNavbar')
+
+		return {
+			// @ts-expect-error
+			isNarrow: computed(() => root.$KtNavbar.isNarrow),
+			title: computed(() => translations.value.quickLinks),
```

Comment:
are u missing a `.title` or am i not grasping the proper type here

Reply:
I guess you only expose the title off the provide anyways - can u please rename the variable in that case to be `quickLinks` on the translations

Reply:
Nope, this is accessing the translations. `quickLinks` are just arrays now, as we never used the `title` prop on the old `quickLinks` object (that was documented as an array anyway...)

Now, this is handled by the `KtTranslationContext`